skynet/modules/ttt/rag/stores/faiss.py

```diff
@@ -62,14 +63,20 @@ async def create(self, store_id, documents):
         index = faiss.IndexFlatL2(len(self.embedding.embed_query(store_id)))
         vector_store = FAISS(
             embedding_function=self.embedding,
-            distance_strategy=DistanceStrategy.EUCLIDEAN_DISTANCE,
+            distance_strategy=DistanceStrategy.COSINE,
```

Comment:
Can we leave a comment at the top of this function about what we are doing and why?